qt-python/src/project.ts

```diff
@@ -1,38 +1,47 @@
 // Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only
 
+import _ from 'lodash';
+import * as fs from 'fs';
+import * as path from 'path';
 import * as vscode from 'vscode';
+import { parse } from 'smol-toml';
 
-import { Project, createLogger } from 'qt-lib';
+import {
+  Project,
+  createLogger,
+  ConfigType,
+  QtWorkspaceConfigMessage,
+  CoreKey,
+  QtWorkspaceFeatures
+} from 'qt-lib';
 import { PySideEnv } from './env';
 import { PySideProjectInfo } from './types';
+import { pyApi, coreApi } from './extension';
+import * as consts from '@/constants';
 
 type Folder = vscode.WorkspaceFolder;
 type Context = vscode.ExtensionContext;
 
 const logger = createLogger('project');
 
 export class PySideProject implements Project {
-  private _env: PySideEnv | undefined;
+  private readonly _env: PySideEnv;
   private _info: PySideProjectInfo | undefined;
 
   private constructor(private readonly _folder: Folder) {
     logger.info(`Create: "${_folder.uri.fsPath}"`);
+    this._env = new PySideEnv(_folder);
   }
 
   dispose() {
     logger.info(`Dispose: "${this._folder.uri.fsPath}"`);
```

Comment:
Can you also dispose `_env` here?
```suggestion
    logger.info(`Dispose: "${this._folder.uri.fsPath}"`);
    this._env.dispose();
```

Reply:
Thanks for noticing that!